bin/node-template/node/src/service.rs

```diff
@@ -72,7 +72,7 @@ macro_rules! new_full_start {
 pub fn new_full(config: Configuration)
 	-> Result<impl AbstractService, ServiceError>
 {
-	let is_authority = config.roles.is_authority();
+	let is_authority = config.role.is_authority();
```

Comment:
Ah that's annoying. @bkchr had a different idea about this. Could you fight together and choose which one you want to use (roles or role)?

Reply:
Before this PR it was called roles since you could have more than one (it was a bitfield), with the changes in this PR you can only have one role.

Reply:
Ah yeah it was written in the description :sweat: